src/config/ios/findProject.js

```diff
@@ -6,21 +7,38 @@ const glob = require('glob');
 const GLOB_PATTERN = '**/*.xcodeproj';
 
 /**
+ * Regexp matching all test projects
+ */
+const TEST_PROJECTS = /(test|example)/;
```

Comment:
Yeah, but if you could make it to be case insensitive- i am happy to remove toLowerCase since it kinda makes the code ugly!